### PR TITLE
Use consul.health.service()

### LINKED ISSUE #12: Replicaset on Joyent - replicas not connecting
So I have MongoDB replicasets running locally on Docker for Mac without using `network: bridge` however, when running on Joyent I can't seem to get the replica to connect and sync with the primary.

1. Pulled down the repo
2. Updated docker-compose.yml ([see gist](https://gist.github.com/sberryman/0c61b319e57de014fc43e8af414a14ee#file-docker-compose-yaml))
3. Started it `docker-compose -p mdb_test -f docker-compose.yml up -d`
4. Waited for consul and mongodb to start and then ran `docker logs mdbtest_mongodb_1` ([see gist](https://gist.github.com/sberryman/0c61b319e57de014fc43e8af414a14ee#file-mongodb_1-log))
5. It took a little white but it managed to create the `mongodb-primary` key and lock session in consul
6. Then I ran `rs.status()` on the only running mongodb node ([see gist](https://gist.github.com/sberryman/0c61b319e57de014fc43e8af414a14ee#file-mdb_rs_status1-js))
7. Now I scaled up by 1 node to have 2 running mongodb nodes. `docker-compose -p mdb_test -f docker-compose.yml scale mongodb=2`
8. Waited for mongodb2 to show up ([see gist](https://gist.github.com/sberryman/0c61b319e57de014fc43e8af414a14ee#file-mongodb_2-log))
9. Ran `rs.status()` on the primary again. ([see gist](https://gist.github.com/sberryman/0c61b319e57de014fc43e8af414a14ee#file-mdb_rs_status2-js))

`docker ps`:
```shell
CONTAINER ID        IMAGE                      COMMAND                  CREATED             STATUS              PORTS                                                                                NAMES
56cedfc2fdbf        autopilotpattern/mongodb   "containerpilot mo..."   5 minutes ago       Up 5 minutes        0.0.0.0:27017->27017/tcp                                                             mdbtest_mongodb_2
2a3303edc766        autopilotpattern/mongodb   "containerpilot mo..."   15 minutes ago      Up 14 minutes       0.0.0.0:27017->27017/tcp                                                             mdbtest_mongodb_1
e5a1f80f8cb1        consul:0.7.5               "docker-entrypoint..."   17 minutes ago      Up 17 minutes       8300-8302/tcp, 8400/tcp, 8301-8302/udp, 8600/tcp, 8600/udp, 0.0.0.0:8500->8500/tcp   mdbtest_consul_1
```

I've tried with and without exposing any ports publicly and with and without using the consul agent. Only thing I can think of at this point is possibly around DNS. I see the redis autopilot pattern is using a very different method to get the IP address of the container.

MongoDB: https://github.com/autopilotpattern/mongodb/blob/master/bin/manage.py#L420-L430
Redis: https://github.com/autopilotpattern/redis/blob/master/bin/manage.sh#L311-L313

I figure I have to be doing something wrong as nobody else has mentioned a problem...

### PR DESCRIPTION
**Pull request by sberryman**
Fixes #12

Also upgrade python dependencies and containerpilot to 2.7.2

### CODE REVIEW

File: bin/manage.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 from pymongo import MongoClient
 from pymongo.errors import *
 
-consul = pyconsul.Consul(host=os.environ.get('CONSUL', 'consul'))
+consul = pyconsul.Consul(host='localhost')
```
Comment (geek) [inline bin/manage.py]: We will likely need to make the consul agent optional in the future, can you revert this change.

**Comment by geek** [comment]
@sberryman LTGM, just one question on manage.py

**Comment by sberryman** [comment]
@geek I've added `CONSUL_AGENT` to the containerpilot.json and updated the manage.py to account for it

**Comment by geek** [comment]
@sberryman thanks for the help, I'll get this released shortly.

**Pull request merged**